composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
Alternately, we could have a `conflict` section:

```json
"require": {
    "psr/container": "^1.0"
},
"conflict: {
    "container-interop/container-interop": "<1.2.0"
}
```

Since the various container implementations we support all require container-interop currently, and version 1.2.0 of container-interop extends the various psr/container interfaces, then an update would grab that version, and the various typehints we use would work.